starter/src/main/java/com/starter/fullstack/dao/InventoryDAO.java

```diff
@@ -1,15 +1,21 @@
 package com.starter.fullstack.dao;
 
-import com.starter.fullstack.api.Inventory;
 import java.util.List;
 import java.util.Optional;
+
 import javax.annotation.PostConstruct;
+
+import org.apache.maven.surefire.shade.org.apache.maven.shared.utils.cli.Arg;
```

Comment:
Remove this import and the following imports:

- `import com.fasterxml.jackson.annotation.JsonTypeInfo.Id;`
- `import com.fasterxml.jackson.core.sym.Name;`

Any more checkstyle errors can be found by running `./gradlew build` in the "starter" directory.